src/gui/shell.h

```diff
@@ -123,6 +124,8 @@ protected slots:
 	void neovimExited(int);
 	void neovimResizeFinished();
 	void mouseClickReset();
+	void localeChanged();
```

Comment:
All new functions should be marked as `noexcept`... Yet another weird C++ thing.